docs/src/contributing.md

```diff
@@ -2,33 +2,65 @@
 
 This guide contains tips and tricks for working on Shotover Proxy itself.
 
-## Building Shotover
+## Configuring the Environment
 
 Shotover is written in Rust, so make sure you have a rust toolchain installed. See [the rustup site](https://rustup.rs/) for a quick way to setup your Rust development environment.
 
 Once you've installed Rust via Rustup (you should just be fine with the latest stable). You will need to install a few other tools needed to compile some of Shotover's dependencies.
 
 Shotover requires the following in order to build:
 
-* cmake
 * gcc
 * g++
 * libssl-dev
 * pkg-config (Linux)
 
-On ubuntu you can install them via `sudo apt-get install cmake gcc g++ libssl-dev pkg-config`
+On Ubuntu you can install them via `sudo apt-get install cmake gcc g++ libssl-dev pkg-config
 
+### Installing Optional Tools and Libraries
+
+#### Docker
 While not required for building Shotover, installing `docker` and `docker-compose` will allow you to run Shotover's integration tests and also build the static libc version of Shotover.
 
+#### libpcap-dev
 Some tests will require `libpcap-dev` to be installed as well (reading pcap files for protocol tests).
 
+## Building Shotover 
+
 Now you can build Shotover by running `cargo build`. The executable will then be found in `target/debug/shotover-proxy`.
 
-## Building Shotover (release)
+### Building Shotover (release)
```

Comment:
This should go above `### Installing Optional Tools and Libraries`

Reply:
ahhh sorry, my bad.
I must of misread a `#` depth level.
What you have is what we want.